test/functional/pdo_sqlsrv/pdo_connect_encrypted_ksp_errors.phpt

```diff
@@ -71,29 +71,29 @@ Connect using a custom keystore provider with some required inputs missing
 
     echo "Done\n";
 ?>
---EXPECT--
+--EXPECTREGEX--
 Connecting... with column encryption
 Connected successfully with ColumnEncryption enabled and KSP specified.
 
```

Comment:
I think you'd need to escape all the periods in the output, they would match up with anything I think.